runtime/config.ts

```diff
@@ -47,7 +66,7 @@ export interface SkyuxConfig {
   cssPath?: string;
   command?: string;
   compileMode?: string;
-  help?: any;
+  help?: HelpWidgetConfig;
```

Comment:
I'm curious about the creation of this interface.  We'd previous used interfaces for the auth and help clients.  We decided against that; however, as somewhat painfully tied those libraries to builder, but without any benefit (consumers never use the interface).

Reply:
That makes sense.  I added it because I was using it for the Help Library and thought builder may want to also make use of it.  That being said, having to update it in multiple places should one change would be a pain, I am fine with removing it.